manifests/f/FarCorp/Far-Anti-Virus/4.31.3102.572/FarCorp.Far-Anti-Virus.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FarCorp.Far-Anti-Virus
+PackageVersion: 4.31.3102.572
+PackageLocale: en-US
+Publisher: Faronics Corporation
+PublisherUrl: https://www.faronics.com/
+PublisherSupportUrl: https://www.faronics.com/company/contact
+PrivacyUrl: https://www.faronics.com/dfc-endpoint-protection-anti-virus
+Author: Faronics
+PackageName: Far-Anti-Virus

```

```suggestion
PackageName: Faronics Anti-Virus
```
This must match Control Panel.